website/index.html

```diff
@@ -9,13 +9,12 @@
       <div class="container">
           <div class="row">
               <div class="col-md-12 text-center">
-                  <h2>Make Your Company Data Driven</h2>
-                  <h3>Connect to any data source, easily visualize and share your data</h3>
+                  <font color="F2F9FC"><h2>Make Your Company Data Driven</h2>
```

Comment:
Before you mentioned I didn't see any difference while comparing between two tabs :-)